source/keyboardHandler.py

```diff
@@ -162,6 +167,21 @@ def internal_keyDownEvent(vkCode,scanCode,extended,injected):
 				return False
 	except:
 		log.error("internal_keyDownEvent", exc_info=True)
+	finally:

```

Comment:
Might be worth commenting why we do this in the finally block; i.e. because we return early in several places but we still want this to be executed.